activerecord/lib/active_record/connection_adapters/mysql2_adapter.rb

```diff
@@ -171,6 +164,13 @@ def translate_exception(exception, message:, sql:, binds:)
             super
           end
         end
+
+        def default_prepared_statements # :nodoc:
```

Comment:
```suggestion
        def default_prepared_statements
```

This is a private method, so it's already undocumented.

Reply:
Ah, yes, I removed it from the other method but forgot this one 🙈